frontend/packages/knative-plugin/src/plugin.tsx

```diff
@@ -31,13 +37,41 @@ const plugin: Plugin<ConsumedExtensions> = [
       models: _.values(models),
     },
   },
+  {
+    type: 'FeatureFlag/Model',
+    properties: {
+      model: models.ConfigurationModel,
+      flag: FLAG_KNATIVE_SERVING_CONFIGURATION,
```

Comment:
`FLAG_KNATIVE_SERVING_CONFIGURATION` we are not using as of now still no harm in having it.

Reply:
I thought it was worth it since we were adding the others. Just for completeness.